test/intl402/NumberFormat/prototype/format/format-rounding-priority-less-precision.js

```diff
@@ -0,0 +1,47 @@
+// Copyright 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-intl.numberformat.prototype.format
+description: >
+  When roungingPriority is "lessPrecision", the constraint which produces the
```

Comment:
Suggestion: Add tests for roundingPriority when all 4 settings are used at the same time (min+max fraction+significant)